covalent/_dispatcher_plugins/local.py

```diff
@@ -82,10 +85,14 @@ def wrapper(*args, **kwargs) -> str:
             json_lattice = lattice.serialize_to_json()
 
             test_url = f"http://{dispatcher_addr}/api/submit"
-
-            r = requests.post(test_url, data=json_lattice)
-            r.raise_for_status()
-            return r.content.decode("utf-8").strip().replace('"', "")
+            try:
+                r = requests.post(test_url, data=json_lattice)
```

Comment:
should we call this something else other than `test_url`? @AlejandroEsquivel @cjao @kessler-frost

Reply:
Could we change it to dispatcher_url?